Makefile

```diff
@@ -41,20 +41,17 @@ build:
 	DOCKER_BUILDKIT=1 docker build --tag $(IMAGE) --target $(DOCKER_IMAGE_TAG)  --build-arg record=$(record) .
 
 build_venv:
-	python3 -m venv mava_env
-	mava_env/bin/pip install git+https://github.com/RuanJohn/jumanji.git@vmap-ippo
-	mava_env/bin/pip install -r https://raw.githubusercontent.com/luchris429/purejaxrl/main/requirements.txt
-	mava_env/bin/pip install jax==0.4.9
-	mava_env/bin/pip install jaxlib==0.4.7+cuda11.cudnn82  -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
-	echo 'export PYTHONPATH="./"' >> mava_env/bin/activate
+	python3 -m venv mava_30
+	mava_30/bin/pip install -r jax_distribution/requirements.txt
+	mava_30/bin/pip install jax==0.4.9
+	mava_30/bin/pip install jaxlib==0.4.7+cuda11.cudnn82  -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
```

Comment:
A problem here is that we are assuming the cuda version someone will have locally. 
Also, is there a reason for the mismatch between the `jax` and `jaxlib` versions?

Reply:
Yes, this is what I was asking about in the previous comment. Should we support multiple builds?

Reply:
maybe we can add a comment here mentioning that you need to update cuda based on what you have

Reply:
I will add a comment here for now. I am not sure what the best way is to address this.